pkg/steps/multi_stage.go

```diff
@@ -34,6 +34,12 @@ const (
 	SecretMountEnv = "SHARED_DIR"
 	// ClusterProfileMountEnv is the env we use to expose the cluster profile dir
 	ClusterProfileMountEnv = "CLUSTER_PROFILE_DIR"
+	// CliMountPath is where we mount the cli in a pod
+	CliMountPath = "/cli"
+	// OcEnv if the env we use to expose the path to the cli
+	OcEnv = "OC"
+	// OcEnvValue is the value of OcEnv
+	OcEnvValue = CliMountPath + "/oc"
```

Comment:
I wonder if we should try harder to place the `oc` to a path with a chance of being in `$PATH`? Exposing the `oc` only via the envvar would block us from seamlessly migrate existing `openshift_installer_src` tests to multi-stage (which I hoped would be possible), because the current workloads are unlikely to use `$OC`